README.md

```diff
@@ -10,13 +10,13 @@ DharmaDB is a persistent, fault tolerant Key-Value Store written in Rust.
 Just create an instance of `Dharma` to get the key value store up and running.
 ```rust
 use dharmadb::dharma::Dharma;
-use dharmadb::errors::Errors;
+use dharmadb::errors::Result;
```

Comment:
```suggestion
use dharmadb::types::Result;
```

Reply:
Can `Result` be moved into a new `types` module since `Result` is not an error?

Reply:
what about a `result` module? I've seen this pattern in other crates; and it fits semantically since an error is a kind of result. What do you think?

Reply:
That sounds great!